### ISSUE TITLE
File upload input type is missing corresponding label

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#2988]
Found while adding axe accessibility testing.

Suggested fixes from axe: https://dequeuniversity.com/rules/axe/4.4/label?application=axeAPI

I think we should add a label and link it to the file input element with IDs, like how we do with FieldWithLabel.
